src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -660,12 +666,7 @@ private void ParseSnippet()
                 if (AtStartOfNonEmpty(_strings.StartLine))
                 {
                     string value = _reader.ReadElementContentAsString();
-                    int.TryParse(value, out startLine);
-                }
-                else if (AtStartOfNonEmpty(_strings.EndLine))
-                {
-                    string value = _reader.ReadElementContentAsString();
-                    int.TryParse(value, out endLine);
+                    int.TryParse(value, out snippetStartLine);
```

Comment:
>int.TryParse(value, out snippetStartLine); [](start = 20, length = 42)

suggest using a mechanism that will throw an exception here rather than swallowing. larry, your opinion>

Reply:
Could just switch to int.Parse

Reply:
Could just switch to int.Parse

---
In reply to: [210471776](https://github.com/Microsoft/sarif-sdk/pull/1020#discussion_r210471776) [](ancestors = 210471776)